Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -518,9 +518,9 @@ open class NavigationMapView: UIView {
             var parentLayer: String? = nil
             let identifiers = [
                 IdentifierString.arrow,
-                IdentifierString.arrowSymbol,
```

Comment:
Original sequence of layer was working correctly in Navigation SDK `1.x`, why this change is required in `2.x`?

Reply:
In Carbon if we're using the 1.x , the adding layer sequence would cause the maneuvers not shown at all. The new sequence of the adding identifiers is following the layer addition and overlapping sequence.